etna/transforms/feature_selection/filter.py

```diff
@@ -18,14 +23,17 @@ def __init__(self, include: Optional[Sequence[str]] = None, exclude: Optional[Se
             list of columns to pass through filter
         exclude:
             list of columns to not pass through
-
+        return_features:
+            bool, indicates whether to return features or not.
```

Comment:
Make the description here consistent with the descriptions in the other files(remove bool)